openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2025 Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def _update_pos_payment_method_journal(env):
+    """From now on, a journal is required when the transactions aren't splitted. If we
+    don't set a journal in these cases, the session closing won't be made properly for
+    these methods. Cash payment methos already had a jornal set, but bank ones didn't."""
+    payment_methods = env["pos.payment.method"].search(
+        [
+            ("journal_id", "=", False),
+            ("split_transactions", "=", False),
+            ("type", "=", "bank"),
+        ]
+    )
+    for method in payment_methods:
+        method.journal_id = env["account.journal"].create(
+            {
+                "type": "bank",
+                "name": f"[openupgrade] Journal for {method.name}",
+                "code": "POS_BANK",
```

Comment:
Hi @chienandalu. Could you fix this point ? https://github.com/OCA/OpenUpgrade/commit/bde6c18907e1daa4a03c00e9bcb6dc34201f18d2#r154676459


(Thanks to @em230418)

Reply:
I will publish a patch son

Reply:
Solved in #4917